### ISSUE TITLE
"set_session cannot be used inside a transaction" after installing SQLAlchemy

### ISSUE THREAD

**Issue by roehst** [dbcli/pgcli#652]
Hi, I used pgcli for a couple months, presumably on top of psycopg2. I installed SQLAlchemy and now pgcli won't even start, and will stop with the following message:

"set_session cannot be used inside a transaction"

What info can I give to help debugging?

Thanks

**Comment by j-bennet** [comment]
What OS are you on?
Can you provide `pip freeze` output?
Are you running `pgcli` in virtualenv?

**Comment by roehst** [comment]
OS: Linux 4.9.11-1-ARCH x86_64 GNU/Linux

Not using virtualenv.

Python version 3.6.0

pip3 freeze

```
appdirs==1.4.2
astroid==1.4.9
attrs==16.3.0
Beaker==1.8.1
beautifulsoup4==4.5.3
Brlapi==0.6.5
chardet==2.3.0
click==6.7
configobj==5.0.6
cupshelpers==1.0
cycler==0.10.0
Cython==0.25.2
decorator==4.0.11
format-sql==0.12.0
greenlet==0.4.11
humanize==0.5.1
ipykernel==4.5.2
ipython==5.1.0
ipython-genutils==0.1.0
ipywidgets==5.2.2
isc==2.0
isort==4.2.5
Jinja2==2.9.5
jsonschema==2.6.0
jupyter-client==4.4.0
jupyter-console==5.1.0
jupyter-core==4.2.1
lazy-object-proxy==1.2.2
louis==3.0.0
Mako==1.0.6
Markdown==2.6.8
MarkupSafe==0.23
matplotlib==2.0.0
mccabe==0.6.1
mistune==0.7.3
msgpack-python==0.4.8
nbformat==4.2.0
neovim==0.1.13
notebook==4.3.1
numpy==1.12.0
packaging==16.8
pandas==0.19.2
pexpect==4.2.1
pgcli==1.4.0
pgspecial==1.7.0
pickleshare==0.7.4
prompt-toolkit==1.0.13
psutil==5.1.3
psycopg2==2.7
ptyprocess==0.5.1
pycups==1.9.73
pycurl==7.43.0
pyenchant==1.6.8
Pygments==2.2.0
pygobject==3.22.0
pygtkspellcheck==4.0.5
pylint==1.6.5
pyparsing==2.1.10
PyQt5==5.8
python-dateutil==2.6.0
python-libtorrent==1.1.1
pytz==2016.10
pyxdg==0.25
pyzmq==16.0.2
ranger==1.8.1
requests==2.13.0
setproctitle==1.1.10
simplegeneric==0.8.1
simplejson==3.10.0
sip==4.19.1
six==1.10.0
SQLAlchemy==1.1.6
sqlparse==0.2.3
team==1.0
termcolor==1.1.0
terminado==0.6
tornado==4.4.2
traitlets==4.3.2
ufonormalizer==0.1
urllib3==1.20
wcwidth==0.1.7
widgetsnbextension==1.2.6
wrapt==1.10.8
```

**Comment by amjith** [comment]
Hi. It is fixed in master. Haven't released it yet. You can install directly from master: 

```
pip install -U https://github.com/dbcli/pgcli/archive/master.zip
```

If you're not comfortable installing directly from master you can downgrade the psycopg2 version to 2.6.2. It's an issue with the latest pscyopg2.

**Comment by j-bennet** [comment]
Fixed by https://github.com/dbcli/pgcli/pull/647.

Available in 1.5.0.
